qiskit/qobj/qasm_qobj.py

```diff
@@ -365,13 +373,18 @@ class QasmQobjExperimentHeader(QobjDictField):
 class QasmQobjExperimentConfig(QobjDictField):
     """Configuration for a single QASM experiment in the qobj."""
 
-    def __init__(self, calibrations=None, **kwargs):
+    def __init__(self, qubit_lo_freq=None, meas_lo_freq=None, calibrations=None, **kwargs):
```

Comment:
Also add new keywords arguments after existing arguments to prevent breaking code that uses positional arguments.

```suggestion
    def __init__(self, calibrations=None, qubit_lo_freq=None, meas_lo_freq=None, **kwargs):
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c.